### PR TITLE
[wasm][sdk]: Add new `Mono.WebAssembly.Runtime.Aot` SDK.

### PR DESCRIPTION
**Pull request by baulig**
* sdks/Mono.WebAssembly.Runtime.Aot: bundle the AOT cross-compiler.

* sdk/Mono.WebAssembly.Runtime.Framework: add a few sources files that are needed by the packager.

### CODE REVIEW

**Comment by kjpou1** [comment]
@baulig I think we will need the zlib extra files as well.

https://github.com/mono/mono/blob/master/sdks/wasm/Makefile#L704
https://github.com/mono/mono/blob/master/sdks/builds/wasm.mk#L11
https://github.com/mono/mono/blob/master/sdks/wasm/packager.cs#L923-L927

They are made available in the `mono/sdks/out/wasm-runtime-release/include/support` directory.

**Comment by baulig** [comment]
@kjpou1 Should we also bump the version?

File: sdks/wasm/sdk/Mono.WebAssembly.Runtime.Framework/Mono.WebAssembly.Runtime.csproj [hunk 1]
```diff
@@ -30,4 +30,15 @@
 
     <None Update="build\netstandard2.0\Mono.WebAssembly.Runtime.props" PackagePath="build\netstandard2.0\Mono.WebAssembly.Runtime.props" Pack="True" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\..\..\wasm\runtime.js" PackagePath="mono-wasm-runtime\src\%(Filename)%(Extension)" />
+    <Content Include="..\..\..\wasm\runtime-tests.js" PackagePath="mono-wasm-runtime\src\%(Filename)%(Extension)" />
+    <Content Include="..\..\..\wasm\src\driver.c" PackagePath="mono-wasm-runtime\src\%(Filename)%(Extension)" />
```
Comment (vargaz) [inline sdks/wasm/sdk/Mono.WebAssembly.Runtime.Framework/Mono.WebAssembly.Runtime.csproj]: runtime-tests.js is only needed for running our test suites.

**Comment by akoeplinger** [comment]
This is no longer needed now that wasm lives in dotnet/runtime.

**Pull request closed without merging**